packages/react/src/ui/InputField/InputField.tsx

```diff
@@ -376,56 +375,62 @@ const InputField = forwardRef<HTMLDivElement, InputFieldProps<string>>(
+                  <F0Icon
+                    onClick={handleClickContent}
+                    icon={icon}
+                    color="default"
+                  />
+                )}
+                {avatar && (
+                  <F0AvatarPerson
+                    firstName={avatar}
+                    lastName={""}
+                    size="xsmall"
+                  />
+                )}
               </div>
             )}
-            <div onClick={handleClickChildren} className="w-full">
-              {cloneElement(children as React.ReactElement, {
-                onChange: handleChange,
-                onBlur: props.onBlur,
-                onFocus: props.onFocus,
-                disabled: noEdit,
-                readOnly: readonly,
-                role,
-                "aria-controls": ariaControls,
-                "aria-expanded": ariaExpanded,
-                id,
-                value: localValue,
-                "aria-label": label || placeholder,
-                "aria-busy": loading,
-                "aria-disabled": noEdit,
-                name,
-                className: cn(
-                  "h-full w-full min-w-0 px-3",
-                  "[&::-webkit-search-cancel-button]:hidden",
-                  icon && "pl-8",
-                  icon && size === "md" && "pl-9",
-                  disabled && "cursor-not-allowed",
-                  (children as React.ReactElement).props.className,
-                  inputElementVariants({ size })
-                ),
-              })}
-            </div>
-            {!noEdit && (
+            {!!localValue && (
+              <div onClick={handleClickChildren} className="w-full">
+                {cloneElement(children as React.ReactElement, {
+                  onChange: handleChange,
+                  onBlur: props.onBlur,
+                  onFocus: props.onFocus,
+                  disabled: noEdit,
+                  readOnly: readonly,
+                  role,
+                  "aria-controls": ariaControls,
+                  "aria-expanded": ariaExpanded,
+                  id,
+                  value: localValue,
+                  "aria-label": label || placeholder,
+                  "aria-busy": loading,
+                  "aria-disabled": noEdit,
+                  name,
+                  className: cn(
+                    "h-full w-full min-w-0 pl-1 pr-3",
+                    "[&::-webkit-search-cancel-button]:hidden",
+                    disabled && "cursor-not-allowed",
+                    (children as React.ReactElement).props.className,
+                    inputElementVariants({ size })
+                  ),
+                })}
+              </div>
+            )}
+            {!noEdit && !localValue && (
               <div
                 className={cn(
-                  "pointer-events-none absolute bottom-0 left-0 top-[1px] z-10 flex flex-1 justify-start px-3 text-f1-foreground-secondary transition-opacity",
```

Comment:
i think that removing the absolute position of the placeholder can affect to the other components position inside the inputfield